Bottom: 87e6f1d871d75ff83a8f1a129dfe089d23db1aa5
Top:    e8d758d88f6cc87ab9412aed162c5813b4d49716
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-02 14:26:57 +0000

Fix make distcheck by removing regenerated newlib footprints

The generated newlib files need to be removed by make distclean
if they are being updated by the build process. If the files aren't
being updated, we should not delete them as they were originally
distributed with the source tarball.


---

diff --git a/lib/Makefile.am b/lib/Makefile.am
index 95be997..ae1b485 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -57,6 +57,11 @@ ${NEWLIB}.stamp : footprint.pcb ${NEWLIB_DEPS}
 		${PREVIEW} --pcb "${PCB}" --awk "${AWK}"
 	echo '${PREVIEW_MSG}' > $@
 
+# Remove any rebuilt footprints
+if BUILD_PCBLIB_NEWLIB
+distclean-local:
+	rm -fr ${NEWLIB}
+endif
 
 install-data-local: ${NEWLIB}.stamp
 	if test -d ${NEWLIB} ; then \
